package.json

```diff
@@ -3,10 +3,8 @@
   "version": "1.0.4",
   "description": "JS utils for interacting with MIT Open Course search",
   "main": "dist/index.js",
-  "module": "src/index.js",
   "files": [
-    "dist/",
-    "src/"
```

Comment:
Not really sure what the consequences are of removing lines 6 and 9, was this on purpose or by accident?

Reply:
I made a change to the url_utils file which used `??`, the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator). I previously was including all of the files in `src` (via this entries in the `files` array) so that ES module aware bundlers like webpack could use the legit ES module instead of the transpiled ES5 code, allowing them to use things like tree shaking and whatnot.

but I was running into an issue where webpack was trying to include `src/url_utils.js` and was throwing an error because of the `??`. I figured it was easier to just stop shipping the `src` dir in the npm package, since the benefit from including it was pretty marginal and just using the ES5 code meets our needs.